prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.html

```diff
@@ -12,31 +12,86 @@
     <section *ngIf="isDeviceProvider"
              class="mb-3">
 
+      <app-page-subheader>
+        <ng-container appPageSubheaderTitle>What is your role?</ng-container>
+        <ng-container appPageSubheaderSummary>
+          Please select “none” to continue if your role is not listed.
+        </ng-container>
+      </app-page-subheader>
+
+      <div class="row">
+        <div class="col col-sm-10 py-3">
+
+          <mat-form-field class="w-100">
+            <mat-label>Your Role</mat-label>
+            <mat-select formControlName="deviceProviderRoleCode">
+              <mat-option *ngFor="let role of deviceProviderRoles"
+                          [value]="role.code">
+                {{ role.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="formState.deviceProviderIdentifier.hasError('required')">
+              Required
+            </mat-error>
+          </mat-form-field>
+        </div>
+      </div>
+
       <app-page-subheader>
         <ng-container appPageSubheaderTitle>Device Provider ID</ng-container>
         <ng-container appPageSubheaderSummary>
-          Provide your Device Provider ID if you have one.
+          Enter the device provider registration number you were issued by the Ministry of Health. You will know it as
+          P1-90XXX (last three characters are numbers).
+        </ng-container>
+      </app-page-subheader>
+
+      <div class="row">
+        <div class="col col-sm-10 py-3">
+          <mat-form-field class="w-100">
+            <mat-label>Device Provider ID</mat-label>
+            <input matInput
+                   maxlength="8"
+                   placeholder=""
+                   formControlName="deviceProviderId">
+            <mat-error *ngIf="formState.deviceProviderId.hasError('deviceProviderId')">
+              Must be in format P1-90XXX (The last three digits are numeric)
```

Comment:
Wondering whether the hint should be "(last three characters are numbers)"?

Reply:
Text updated as suggested